### ISSUE TITLE
aria-required-children can double-report related nodes through aria-owns

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#4840]
In the following example `#foo` is reported twice as a related element, once for being a child, and once for aria-owns.

```html
<div role="tablist" aria-owns="foo">
  <div id="foo" role="menuitem">foo</div>
</div>
```
